site/content/examples/15-composition/04-modal/Modal.svelte

```diff
@@ -2,7 +2,23 @@
 	import { createEventDispatcher } from 'svelte';
 
 	const dispatch = createEventDispatcher();
-</script>
+	
+	const handleClose = () => dispatch('close')
+
+	function handleKeydown(event) {
+		if (event.key == 'Escape') {
+			handleClose()
+		} else if (event.key == 'Tab') {
+			event.preventDefault()
+		}
+	}
+
+	let closeButton
+	onMount(() => {
+		closeButton.focus()
+	})
+
+</cript>
```

Comment:
Looks like a typo here with a missing s at the start of "</script>"
```suggestion
</script>
```

Reply:
yikes! you are correct. Thanks